charts/incubator/clickhouse/values.yaml

```diff
@@ -0,0 +1,70 @@
+image:
+  repository: yandex/clickhouse-server
+  pullPolicy: IfNotPresent
+  tag: 21.3.20.1@sha256:4eccfffb01d735ab7c1af9a97fbff0c532112a6871b2bb5fe5c478d86d247b7e
+
+controller:
+  type: statefulset
+  strategy: RollingUpdate
+  rollingUpdate:
+    unavailable: 1
+
+securityContext:
+  capabilities:
+    add:
+      - IPC_LOCK
+      - NET_ADMIN
+      - SYS_NICE
+
+service:
+  main:
+    ports:
+      main:
+        port: 8123
+        targetPort: 8123
+
+volumeClaimTemplates:
+  data:
+    enabled: true
+    mountPath: /var/lib/clickhouse
+  config-server:
+    enabled: true
+    mountPath: /etc/clickhouse-server/config.d
+  config-users:
+    enabled: true
+    mountPath: /etc/clickhouse-server/users.d
+  logs:
+    enabled: true
+    mountPath: /var/log/clickhouse-server
+    type: emptyDir
+
+probes:
+  liveness:
+    type: HTTP
+    path: /ping
+  readiness:
+    type: HTTP
+    path: /ping
+  startup:
+    enabled: false
```

Comment:
Please do not disable startup probe. There is no reason to, as it's just another probe level.
If startup probe fails on startup, the others would also fail anyway.

Reply:
I'm just testing and try to replicate the (working) behavior from my test cluster.